templates/packages/order_edit.html

```diff
@@ -0,0 +1,254 @@
+                                            <input type="hidden" name="update-type" value="update"/>
+                                            <button type="submit" class="btn btn-outline-success">Update</button>
+                                        </form>
+                                    </li>
+                                    <li class="list-inline-item me-4">
+                                        <form method="post"> {% csrf_token %}
+                                            <input type="hidden" name="package" value="{{package.id}}" />
+                                            <input type="hidden" name="update-type" value="revert"/>
+                                            <button class="btn btn-outline-danger">Revert</button>
+                                        </form>
+                                    </li>
+                                </ul>
+                            {%endif%}
+                        </p>
+                    </div>
+                </div>
+            </div>
+        {%endfor%}
+    </div>
+    <style>
+        input::-webkit-outer-spin-button,
+        input::-webkit-inner-spin-button {
+           -webkit-appearance: none;
+           margin: 0;
+        }
+        input[type="number"] {
+           -moz-appearance: textfield;
+        }
+    </style>
+    <div class="modal fade" id="package-modal" tabindex="-1" data-bs-backdrop="static" data-bs-keyboard="false" role="dialog" aria-labelledby="package-modal-title" aria-hidden="true" dir="ltr">
+        <div class="modal-dialog modal-dialog-centered">
+            <div class="modal-content">
+                <div class="modal-header" dir="rtl">
+                    <div class="container">
+                        <div class="row">
+                            <div class="col-auto">
+                                <h5 class="modal-title" id="package-modal-title">יצירה/עריכה של חבילה</h5>
+                            </div>
+                            <div class="col">
+
+                            </div>
+                            <div class="col-auto">
+                                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="modal-body">
+                    <div class="container-fluid">
+                        <form id="package-form" method="post"> {%csrf_token%}
+                            <input id="package-id" type="hidden" name="package_id"/>
+                            <label for="origin" class="form-label">Origin</label>
+                            <div id="origin" class="row g-1 mb-3">
+                                <div class="col-4">
+                                    <input id="origin-street" type="text" class="form-control rounded-0 border-0" name="origin_street" list="streets" placeholder="Street"/>

```

Comment:
In reality, separating the address so much is a bit.... meh.
An address is composed of related data, it might be a good idea to look into making it one long smart autocomplete field, like the one you would see in Moovit or Google Maps.

Reply:
We can also go one step further by inserting a google maps auto complete and parsing it on the server side. What do you think?

Reply:
Maybe. Just remember it for later.